upload/modules/CraftingStore/module.php

```diff
@@ -5,37 +5,37 @@ class CraftingStoreModule extends Module
     /**
      * @var InstallDatabaseFlow
```

Comment:
This comment is obsolete if you add a type, could you remove those docs for the items?